components/tyk-bootstrap/templates/bootstrap-post-install.yaml

```diff
@@ -80,11 +81,11 @@ spec:
             - name: OPERATOR_SECRET_NAME
               value: "tyk-operator-conf"
             {{- end }}
-            {{- if .Values.bootstrap.enterprisePortalSecret }}
+            {{- if .Values.global.components.enterprisePortal }}
```

Comment:
Now portal secret will always be created right?
Previously, `bootstrap.enterprisePortalSecret` determined is secret needs to be created or not.

Reply:
Changed to  .Values.global.bootstrap.enterprisePortal